### DISCUSSION TITLE
Discussion categories

### DISCUSSION THREAD

**Discussion by adamcameron** [wheels-dev/wheels#1193] [Ideas]
OK, this will be the last one. For now ;-)

There are discussion categories for: General, Ideas, Polls, Q&A, Show and Tell and General.

No mention of "Bugs", which don't really fit into any of those, and I should think in a GitHub discussion, one of the more common situations?

I mean one could immediately go raise an issue, but in my experience I'm seldom 100% confident what I am seeing is a bug or just me misunderstanding something, or "yeah it's like that by design" (for some definition of "design"), so it's good to have a discussion first, which might result in an issue being raised?

Cheers.

-- 
Adam

**Comment by dbelanger** [comment]
In my opinion, the "Issues" category is very similar to this discussion tab and allows for a discussion.  I think it's healthy to have those conversations there because if it turns out not to be a bug, we can simply close the issue.  If it really is a bug, we have the whole discussion right there and we can always update the issue description after the fact with what we've learned in the discussion.

**Reply by adamcameron** [reply]
Yeah, good thinking. I'm down with that.

**Comment by bpamiri** [comment]
Also, GitHub Issues and Discussions are pretty nebulous. An issue can be changed into a discussion and vise versa. A Discussion can be promoted to an Issue. Movement in either direction retains all the conversation as well.
